harper-core/dictionary.dict

```diff
@@ -20082,10 +20082,9 @@ diaereses/1
 diaeresis/1M
 diagnose/4DSG
 diagnosis/14M
-diagnostic/51S
+diagnostic/51MS
```

Comment:
`5` means proper noun, which was wrong before this change, which is otherwise right.